src/Promenade.Web/Views/Home/LocationDetails.cshtml

```diff
@@ -0,0 +1,214 @@
+                                        </td>
+                                    </tr>
+                            </text>
+                            }
+                            <tr>
+                                <td colspan="2">&nbsp;</td>
+                                <td>
+                                    <a href="/subscribe/?@foreach (var location in Model.NearByLocations) {<text>list=@location.SubscriptionLinkId&</text> }" class="btn btn-default btn-sm" target="_blank">
+                                        <span class="fa fa-envelope-o"></span> Updates from all @Model.NearByLocations.Count() libraries
+                                    </a>
+                                </td>
+
+                            </tr>
+
+                        </tbody>
+
+                    </table>
+
+                </div>
+
+            </div>
+
+        </div>
+        }
+    </div>
+    <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
+        <div class="modal-dialog modal-dialog-centered" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title" id="modalTitle">Details</h5>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">&times;</span>
+                    </button>
+                </div>
+                <div class="modal-body">
+                    <div>
+                        <p id="modal-body">
+                        </p>
+                    </div>
+                    <div>
+                        <p id="modal-text">
+
+                        </p>
+                    </div>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-primary" data-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+    @section Scripts{
+        <script>
+                $('.bring-modal').on('click', function () {
+                    @foreach (var feature in Model.LocationFeatures)
+                    {
+                        <text>
+                        if ("@feature.Stub" == $(this).data("feature"))
+                        {
+                            $("#modalTitle").text("@feature.Name");
+                            $("#modal-body").text("@feature.BodyText");
+                            $("#modal-text").text("@feature.Text");
+                        }
+                        </text>
+                    }
+
+                    $('#myModal').on('shown.bs.modal', function () {
+                        $('#myInput').trigger('focus')
+                    });
+                });
+        </script>
+        <script type="application/ld+json">
```

Comment:
Let's take this `ld+json` out for now until we figure out how to best represent it.